### ISSUE TITLE
https://primeng.org/stepper linear - stackblitz not working because of wrong file name 

### ISSUE THREAD

**Issue by DanielOberlechner** [primefaces/primeng#16967]
### Describe the bug

https://primeng.org/stepper

Open linear example in stackblitz, and it won't load because the stepper-linear-demo.ts has following content:
```ts
import { Component } from '@angular/core';
import { ImportsModule } from './imports';
@Component({
    selector: 'stepper-linear-demo-demo',
    templateUrl: './stepper-linear-demo-demo.html',
    standalone: true,
    imports: [ImportsModule]
})
export class StepperLinearDemo {
}
```

correct would be:
'./stepper-linear-demo.html',
'stepper-linear-demo'

### Environment

stackblitz environemnt

### Reproducer

https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.html

### Angular version

18

### PrimeNG version

17.18.12

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

node 20

### Browser(s)

Chrome newest version on macOS

### Steps to reproduce the behavior

Open stackblitz example and it won't load because of wrong names of the files.

### Expected behavior

The website should load and show example of linear stepper

**Comment by DanielOberlechner** [comment]
I fixed the issue on stackblitz and forked the reposoitory:
https://stackblitz.com/edit/ab7s5gpm-q1eqbneh?file=src%2Fapp%2Fstepper-linear-demo.ts

But not sure how to implement it now on the website. I'm in contact with people on Discord maybe someone will guide me into the right direction.

**Comment by Mickael-Martinez** [comment]
If I understand correctly, the StackBlitz is directly generated from the code.

In this case, the problem comes from line 135 in this file:

https://github.com/primefaces/primeng/blob/17.18.12/src/app/showcase/doc/stepper/lineardoc.ts

`templateUrl: './stepper-linear-demo-demo.html',` (note the double "demo")

**Comment by DanielOberlechner** [comment]
You are right, due to githubs linting I wasn't able to spot the problem right away, I searched before in the right file, but coulnd't find it. I'll fork the repo and create a pull request. Thank you for guiding me into the right direction!
